data/operators/landuse/industrial.json

```diff
@@ -16,6 +16,19 @@
         "operator:wikidata": "Q2265496"
       }
     },
+    {
+      "displayName": "Lenarcik",
+      "id": "lenarcik-9ff3f0",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "brand": "Lenarcik",
```

Comment:
Maybe brand is not necessary here?

Reply:
shop (one of 54): https://www.openstreetmap.org/way/866692988
plant: https://www.openstreetmap.org/way/866693458

product with logo: https://zmlenarcik.pl/wp-content/uploads/2019/10/smalec_dworski.jpg

I think it is brand, but not all shops are named the same, so then also operator